pkg/graph/resolvers/system_intake.go

```diff
@@ -342,7 +342,11 @@ func SystemIntakesWithReviewRequested(ctx context.Context, store *storage.Store)
 }
 
 func GetMySystemIntakes(ctx context.Context, store *storage.Store) ([]*models.SystemIntake, error) {
-    p := appcontext.Principal(ctx)
+p := appcontext.Principal(ctx)
+userAccount := p.Account()
+if userAccount == nil {
+    return nil, fmt.ErrorOf(" there is no user account present")
+}
     if p == nil || p.Account() == nil || p.Account().ID == uuid.Nil {
```

Comment:
thanks Gary, note, the check below should also be removed, you only need to check if the account is nil